Update console.asciidoc note transform GET with body to POST (#108265)
elastic/kibana: docs/dev-tools/console/console.asciidoc

Once copied, the username and password will need to be provided
for the calls to work from external environments.

> Note: If you post a GET request with a body, will be automaically be converted to *Console* syntax as a POST request.

Reviewer: ```suggestion
NOTE: If you post a GET request with a body, it is automatically converted to *Console* syntax as a POST request.
```
